src/frontend/Sider/Behandling/BehandlingTabsInnhold.tsx

```diff
@@ -43,10 +46,16 @@ const BehandlingTabsInnhold = () => {
     const { settToast } = useApp();
     const { behandling, behandlingErRedigerbar, kanBehandleRevurdering } = useBehandling();
 
+    const revurderingFraDatoEnabled = useFlag(Toggle.REVURDERING_FRA_DATO);
+
     const path = useLocation().pathname.split('/')[3];
     const [statusPåVentRedigering, settStatusPåVentRedigering] = useState(false);
 
-    const aktivFane = isFanePath(path) ? path : FanePath.INNGANGSVILKÅR;
+    const førsteFanePath =
+        behandling.type === BehandlingType.FØRSTEGANGSBEHANDLING
```

Comment:
Her må man sjekke på at featureToggle også er enabled, ettersom fanen ellers ikke finnes
```suggestion
        behandling.type === BehandlingType.FØRSTEGANGSBEHANDLING && revurderingFraDatoEnabled
```

Reply:
Nice catch! Fiksa